packages/@aws-cdk/aws-stepfunctions-tasks/lib/emrcontainers/start-job-run.ts

```diff
@@ -0,0 +1,710 @@
+      assumedBy: new iam.ServicePrincipal('emr-containers.amazonaws.com'),
+    });
+
+    // allow job-execution-role to be used by Step Functions
+    jobExecutionRole.assumeRolePolicy?.addStatements(
+      new iam.PolicyStatement({
+        actions: ['sts:AssumeRole'],
+        principals: [new iam.ServicePrincipal('states.amazonaws.com')],
+      }),
+    );
+
+    jobExecutionRole.addManagedPolicy(
+      new iam.ManagedPolicy(this, 'Emr Containers Managing Virtual Clusters Policies', {
+        statements: [
+          new iam.PolicyStatement({
+            resources: ['*'],
+            actions: [
+              'emr-containers:CreateVirtualCluster',
+              'emr-containers:DeleteVirtualCluster',
+              'emr-containers:DescribeVirtualCluster',
+              'emr-containers:ListVirtualClusters',
+            ],
+          }),
+        ],
+      }),
+    );
+
+    jobExecutionRole.addManagedPolicy(
+      new iam.ManagedPolicy(this, 'Emr Containers S3 and Log Policies', {
+        statements: [
+          new iam.PolicyStatement({
+            resources: ['*'],
+            actions: [
+              's3:PutObject',
+              's3:GetObject',
+              's3:ListBucket',
+            ],
+          }),
+          new iam.PolicyStatement({
+            resources: ['arn:aws:logs:*:*:*'],
+            actions: [
+              'logs:PutLogEvents',
+              'logs:CreateLogStream',
+              'logs:DescribeLogGroups',
+              'logs:DescribeLogStreams',
+            ],
+          }),
+        ],
+      }),
+    );
+
+    jobExecutionRole.addManagedPolicy(
+      new iam.ManagedPolicy(this, 'Emr Containers Job Run Policy', {
+        statements: [
+          new iam.PolicyStatement({
+            resources: ['*'],
+            actions: [
+              'emr-containers:StartJobRun',
+              'emr-containers:ListJobRuns',
+              'emr-containers:DescribeJobRun',
+              'emr-containers:CancelJobRun',
+            ],
+          }),
+        ],
+      }),
+    );
+
+    jobExecutionRole.addManagedPolicy(
+      new iam.ManagedPolicy(this, 'Emr Containers Monitoring Policy', {
+        statements: [
+          new iam.PolicyStatement({
+            resources: ['*'],
+            actions: [
+              'emr-containers:DescribeJobRun',
+              'elasticmapreduce:CreatePersistentAppUI',
+              'elasticmapreduce:DescribePersistentAppUI',
+              'elasticmapreduce:GetPersistentAppUIPresignedURL',
+            ],
+          }),
+          new iam.PolicyStatement({
+            resources: ['*'],
+            actions: [
+              's3:GetObject',
+              's3:ListBucket',
+            ],
+          }),
+          new iam.PolicyStatement({
+            resources: ['*'],
+            actions: [
+              'logs:Get*',
+              'logs:DescribeLogGroups',
+              'logs:DescribeLogStreams',
+            ],
+          }),
```

Comment:
duplicated from above

Reply:
Removed because redundant and out of scope